src/components/state-details.js

```diff
@@ -169,11 +98,11 @@ const StateDetailsPage = ({location, data}) => {
       <div className='col-12'>
         <h1 className='display-4 d-flex align-items-center mr-4 mb-3 font-weight-bold'>
           <span className={ 'display-2 mr-4 sf-' + stateFaceClass } aria-hidden="true"></span>
-          {placeData.name}
+          { placeTitle }
         </h1>
       </div>
 
-      { !placeData.emissions }
+      {/* { !placeData.emissions } */}
```

Comment:
Assuming this is for conditional rendering of sections, can be simplified to access `buildingsEmissions`, `dirtyPowerEmissions` and `transportionEmissions` directly!